client/src/models/populate_stores.js

```diff
@@ -260,7 +260,6 @@ function populate_programs(rows) {
       crso,
       activity_code: row[activity_code],
       dept: Dept.lookup(row[dept_code]),
-      data: {},
```

Comment:
I couldn't find this data actually being populated / used but it might've been just that I couldn't find it. If anyone knows we actually need this, let me know